tests/Doctrine/Tests/Common/Proxy/VariadicTypeHintClass.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Doctrine\Tests\Common\Proxy;
+
+/**
+ * Test asset class
+ */
+class VariadicTypeHintClass
+{
+    /**
+     * @param ...$types
+     */
+    public function addType(...$types)
+    {
```

Comment:
Relevant for the proxy logic test (just to test that a method was called with the right amount of args)